$OpenBSD: patch-cmake_modules_FindSqlite_cmake,v 1.1 2017/05/22 19:38:59 espie Exp $
found independently, but confirmed by 
https://git.reviewboard.kde.org/r/119762/

cmake_flags syntax is tricky. and we need to explicitly ask for 
/usr/local/include with clang

Index: cmake/modules/FindSqlite.cmake
--- cmake/modules/FindSqlite.cmake.orig
+++ cmake/modules/FindSqlite.cmake
@@ -94,7 +94,7 @@ if(EXISTS ${SQLITE_INCLUDE_DIR}/sqlite3.h)
                 ${CMAKE_BINARY_DIR}/sqlite_check_unlock_notify
                 ${CMAKE_BINARY_DIR}/sqlite_check_unlock_notify.cpp
                 LINK_LIBRARIES ${SQLITE_LIBRARIES}
-                CMAKE_FLAGS INCLUDE_DIRECTORIES ${SQLITE_INCLUDE_DIR})
+                CMAKE_FLAGS -DINCLUDE_DIRECTORIES:PATH=${SQLITE_INCLUDE_DIR})
     if (NOT SQLITE_HAS_UNLOCK_NOTIFY)
         message(STATUS "Sqlite ${SQLITE_VERSION} was found, but it is not compiled with -DSQLITE_ENABLE_UNLOCK_NOTIFY")
     endif()
